.github/workflows/rolling-release.yml

```diff
@@ -47,242 +51,204 @@ jobs:
-      shell: cmd
-      run: |
-        del /Q bin\ARM64\*.pdb
-        del /Q bin\ARM64\*.exp
-        del /Q bin\ARM64\*.lib
-        del /Q bin\ARM64\*.iobj
-        del /Q bin\ARM64\*.ipdb
-        del /Q bin\ARM64\common-tests*
-        del /Q bin\ARM64\duckstation-libretro-*
-        rename bin\ARM64\updater-ARM64-ReleaseLTCG.exe updater.exe
-                
     - name: Create arm64 release archive
-      shell: cmd
       run: |
-        "C:\Program Files\7-Zip\7z.exe" a -r duckstation-windows-arm64-release.zip ./bin/ARM64/*
+        7z a  duckstation-windows-arm64-release.zip .\bin\ARM64\ -r "-xr!*.pdb" "-xr!common-tests*"
+        7z rn duckstation-windows-arm64-release.zip updater-ARM64-ReleaseLTCG.exe updater.exe
 
     - name: Upload arm64 release artifact
       uses: actions/upload-artifact@v1
       with:
-        name: "windows"
-        path: "duckstation-windows-arm64-release.zip"
+        name: windows-arm64
+        path: duckstation-windows-arm64-release.zip
 
 
-  windows-libretro-build:
+  build-windows-libretro:
     runs-on: windows-2019
+    env:
+      POWERSHELL_TELEMETRY_OPTOUT: 1
     steps:
-    - uses: actions/checkout@v2.3.1
+    - uses: actions/checkout@v2
       with:
         fetch-depth: 0
 
     - name: Compile release build
       shell: cmd
       run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
         mkdir build
         cd build
+        call "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
         cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DBUILD_LIBRETRO_CORE=ON -DCMAKE_C_COMPILER:FILEPATH="%VCToolsInstallDir%\bin\HostX64\x64\cl.exe" -DCMAKE_CXX_COMPILER:FILEPATH="%VCToolsInstallDir%\bin\HostX64\x64\cl.exe" ..
-        ninja
+        ninja -k0
         
     - name: Create libretro core archive
-      shell: cmd
-      run: |
-        cd build
-        "C:\Program Files\7-Zip\7z.exe" a -r duckstation_libretro.dll.zip ./duckstation_libretro.dll
+      run: 7z a -r build\duckstation_libretro.dll.zip .\build\duckstation_libretro.dll
 
     - name: Upload release artifact
       uses: actions/upload-artifact@v1
       with:
-        name: "windows-libretro-x64"
-        path: "build/duckstation_libretro.dll.zip"
+        name: windows-libretro-x64
+        path: build\duckstation_libretro.dll.zip
 
 
-  linux-build:
+  build-linux:
     runs-on: ubuntu-18.04
     steps:
-    - uses: actions/checkout@v2.3.1
+    - uses: actions/checkout@v2
       with:
         fetch-depth: 0
 
     - name: Install packages
-      shell: bash
       run: |
         sudo apt-get update
-        sudo apt-get -y install cmake ninja-build ccache libsdl2-dev libgtk2.0-dev qtbase5-dev qtbase5-dev-tools qtbase5-private-dev qt5-default qttools5-dev
+        sudo apt-get -y install cmake ninja-build ccache {lib{sdl2,gtk2.0},qttools5}-dev qtbase5-{dev{-tools,},private-dev} qt5-default
```

Comment:
Would recommend keeping this explicit for clarity rather than using brace expansion.

Reply:
+1 to this - I prefer the clarity.